--- hamster-applet-0.6.1/hamster/Makefile.am~	2008-06-09 00:45:20.000000000 +0200
+++ hamster-applet-0.6.1/hamster/Makefile.am	2008-07-01 15:53:01.608398281 +0200
@@ -16,12 +16,12 @@
 idle_la_CFLAGS = $(XSCRNSAVER_CFLAGS) $(PYTHON_INCLUDES)
 
 hamster-applet: hamster-applet.py
-	sed -e "s|\@PYTHONDIR\@|$(pythondir)/hamster|" $< > $@
+	sed -e "s|\@PYTHONDIR\@|$(pyexecdir)/hamster|" $< > $@
 
 hamsterbindir = $(libdir)/hamster-applet
 hamsterbin_SCRIPTS = hamster-applet
 
-hamsterdir = $(pythondir)/hamster
+hamsterdir = $(pyexecdir)/hamster
 hamster_PYTHON = \
 	defs.py \
 	dispatcher.py \
--- hamster-applet-0.6.1/hamster/keybinder/Makefile.am~	2008-04-25 00:57:10.000000000 +0200
+++ hamster-applet-0.6.1/hamster/keybinder/Makefile.am	2008-07-01 16:01:05.784232502 +0200
@@ -1,6 +1,6 @@
 INCLUDES = $(PYTHON_INCLUDES) $(HAMSTER_CFLAGS)
 
-_keybinderdir = $(pythondir)/hamster/keybinder
+_keybinderdir = $(pyexecdir)/hamster/keybinder
 _keybinder_LTLIBRARIES = _keybinder.la
 _keybinder_la_LDFLAGS = -module -avoid-version -export-symbols-regex init_keybinder
 _keybinder_la_LIBADD = $(HAMSTER_LIBS)
